### PR TITLE
FLUID-6288 Add Dockerfile

### PR DESCRIPTION
**Pull request by gtirloni**
How to test:

```
$ docker build -t websavvy .
$ docker run --rm -ti -p 8000:80 websavvy
$ open http://localhost:8000
```

### CODE REVIEW

**Comment by gtirloni** [comment]
@jobara @jhung @waharnum @cindyli do you have merge permission in this repository? Could you review and merge this PR? Thanks!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
...
+## Create Docker image
+
+You can serve the website from a [Docker](https://docs.docker.com/get-docker) container.
+
+Once you have Docker installed, run the following commands to build a Docker image and start a container:
+
+* Build the image: `docker build -t websavvy .`
+* Run the container: `docker run --name websavvy -p 8000:80 websavvy`
+
+The website will be available at [http://localhost:8000](http://localhost:8000)
```
Comment (jobara) [inline README.md:12]: would you be able to add the documentation for removing the docker image?

**Comment by greatislander** [comment]
Hi all, this may not be needed anymore as in conversation with @lliskovoi we agreed that the standalone WebSavvy site should be shut down and redirect to the consulting page on the new IDRC site. This task isn't done yet (https://github.com/inclusive-design/idrc/issues/35) but just wanted to flag before you spend too much time on this.

**Comment by jobara** [comment]
@greatislander thanks for that info. @gtirloni what do you think about closing this PR and archiving this repo?

**Comment by gtirloni** [comment]
@jobara sounds good to me, thanks for the heads up

**Pull request closed without merging**